### PR TITLE
Kadence hooks

### PR DESCRIPTION
**Pull request by Camwyn**
This adds a collection of hooks to the single event templates per the ticket.

[TEC-3948]

[TEC-3948]: https://theeventscalendar.atlassian.net/browse/TEC-3948

### CODE REVIEW

**Review by borkweb — DISMISSED** [review]
Just needs a changelog!

File: src/views/single-event-blocks.php [hunk 1]
```diff
@@ -31,7 +31,9 @@
 	<?php if ( $is_recurring ) { ?>
 		<?php $this->template( 'single-event/recurring-description' ); ?>
 	<?php } ?>
+	<?php do_action( 'tribe_events_single_event_block_before_the_content' ) ?>
```
Comment (bordoni) [inline src/views/single-event-blocks.php:34]: In this case here they can use the FIlters from the template system.

File: src/views/single-event-blocks.php [hunk 2]
```diff
@@ -31,7 +31,9 @@
...
 	<?php $this->template( 'single-event/content' ); ?>
+	<?php do_action( 'tribe_events_single_event_block_after_the_content' ) ?>
```
Comment (bordoni) [inline src/views/single-event-blocks.php:36]: In this case here they can use the FIlters from the template system.

File: src/views/v2/default-template.php [hunk 3]
```diff
@@ -15,5 +15,25 @@
...
 get_header();
+
+/**
+ * Provides an action that allows for the injection of HTML at the top of the template after the header.
+ *
+ * @since TBD
+ *
+ * @param string $eventDisplay The string representation (slug) of the displayed view - "month".
+ */
+do_action( 'tribe_default_events_views_v2_template_after_header', $eventDisplay );
```
Comment (bordoni) [inline src/views/v2/default-template.php:26]: Event Display is not a variable in V2. And on this case we can use a entry point here.

File: src/views/v2/default-template.php [hunk 4]
```diff
@@ -15,5 +15,25 @@
...
 echo tribe( Template_Bootstrap::class )->get_view_html();
+
+/**
+ * Provides an action that allows for the injections of HTML at the bottom of the template before the footer.
+ *
+ * @since TBD
+ *
+ * @param string $eventDisplay The string representation (slug) of the displayed view - "month".
+ */
+do_action( 'tribe_default_events_views_v2_template_before_footer', $eventDisplay );
```
Comment (bordoni) [inline src/views/v2/default-template.php:37]: Event Display is not a variable in V2. And on this case we can use a entry point here.

**Review by bordoni — CHANGES_REQUESTED** [review]

File: src/views/v2/default-template.php (hunk 3 above)
Comment (Camwyn) [reply src/views/v2/default-template.php]: Changed the param, but entry_point isn't available here.

File: src/views/v2/default-template.php (hunk 4 above)
Comment (Camwyn) [reply src/views/v2/default-template.php]: Changed the param, but entry_point isn't available here.

**Review by bordoni — APPROVED** [review]

**Pull request merged**